pytest.ini

```diff
@@ -7,6 +7,8 @@ addopts =
     --ignore=dateparser/calendars/hijri_parser.py
     --ignore=dateparser/calendars/hijri.py
 flake8-ignore =
+    W503
```

Comment:
Maybe include a comment to indicate why it is ignored?

Reply:
Done!